### ISSUE TITLE
Is a new release possible?

### ISSUE THREAD

**Issue by ddormer** [sachinraja/rodemirror#73]
Would it be possible to release a newer version of Rodemirror on npmjs? This would solve the error below with the switch to tsup and the react-import injection.

When bundling my project with webpack@5.58.2 and using rodemirror@1.6.3 I get the following error:

```
ERROR in ./node_modules/rodemirror/dist/index.js 1:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/ddormer/code/test-rodemirror/node_modules/rodemirror/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

Adding the `.js` extension to the `react/jsx-runtime'` import in the `node_modules/rodemirror/dist/index.js` file fixes this problem, however it's also fixed in the latest master branch.

**Comment by sachinraja** [comment]
Fixed in [v1.6.4](https://github.com/sachinraja/rodemirror/releases/v1.6.4). Tested against the repro and it worked, again thank you for making it as simple as possible.

**Comment by ddormer** [comment]
@sachinraja Awesome, thanks for taking a look so quickly and for the new release 👍
